conf/core/Testing.conf

```diff
@@ -1,5 +1,6 @@
 {
   "#" : "Options required for unit test runs",
   "hash.seed.zero": "zero",
+  "building.part.merger.thread.count": "1",
   "#" : "end"
-}
\ No newline at end of file
+}
```

Comment:
Decided to just keep one test in BuildingPartMergeOp for testing it running multithreaded.  Will keep all other tests single threaded.